CoffeeChat/Cells/InterestsTableViewCell.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  InterestsTableViewCell.swift
+//  CoffeeChat
+//
+//  Created by Phillip OReggio on 2/3/20.
+//  Copyright © 2020 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+class InterestsTableViewCell: UITableViewCell {
```

Comment:
Lol, I don't know why this is here because I rebased with master and it's not in my project file anymore..